staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -943,6 +943,8 @@ type PodSecurityPolicySpec struct {
 	// is allowed in the "Volumes" field.
 	// +optional
 	AllowedFlexVolumes []AllowedFlexVolume `json:"allowedFlexVolumes,omitempty" protobuf:"bytes,18,rep,name=allowedFlexVolumes"`
+	// runAsGroup is the strategy that will dictate the allowable RunAsGroup values that may be set.
```

Comment:
add `+optional`

Reply:
didnt know about that, will add, is there some documentation about this ? i though omitempty would handle that ?